README.md

```diff
@@ -1,8 +1,10 @@
 # Fivetran Async Provider for Apache Airflow
 
-This package provides an async sensor, and hook that integrates [Fivetran](https://fivetran.com) into Apache Airflow.
+This package provides an async operator, sensor and hook that integrates [Fivetran](https://fivetran.com) into Apache Airflow.
 `FivetranSensorAsync` allows you to monitor a Fivetran sync job for completion before running downstream processes.
-Since an async sensor frees up worker slot while polling is happening on the trigger, it consumes less resources when compared to traditional "sync" sensors.
+`FivetranOperatorAsync` submits a Fivetran sync job and polls for its status on the trigger.
```

Comment:
```suggestion
`FivetranOperatorAsync` submits a Fivetran sync job and polls for its status on the triggerer.
```

Reply:
Fixed in [a430049](https://github.com/astronomer/airflow-provider-fivetran-async/pull/8/commits/a4300496e12aeba5d6673638067638d8ecdee9cc)